### PR TITLE
NFT images

### PR DESCRIPTION
**Pull request by emishur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,7 +207,47 @@ owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 0	balance: 0
 owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 1	balance: 0
 ```
 
-## Transferring Tokens
+### Tokens With External Metadata
+
+Token metadata can store a reference to some external document and/or image.
+This tutorial supports storing external data on [IPFS](https://ipfs.io) and keeping
```
Comment (obstropolos) [inline README.md:213]: keeping -> keeping an

File: README.md [hunk 2]
```diff
@@ -207,7 +207,47 @@ owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 0	balance: 0
...
+IPFS hash as a part of the token metadata.
+
+Let's create a single NFT token which references an image on IPFS.
+
+1. Upload your image to IPFS and obtain image file hash. There are
```
Comment (obstropolos) [inline README.md:218]: obtain -> obtain an

File: README.md [hunk 3]
```diff
@@ -207,7 +207,47 @@ owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 0	balance: 0
...
+   multiple ways to do that. One of the possible solutions is to install
```
Comment (obstropolos) [inline README.md:219]: install -> install the

File: README.md [hunk 4]
```diff
@@ -207,7 +207,47 @@ owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 0	balance: 0
...
+   [IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion) web plugin and
+   upload an image file from there. You can upload multiple images and/or documents
+   if you plan to create a collection of multiple NFTs.
+
+2. Copy IPFS file hash code (`CID`). For this example we will use
```
Comment (obstropolos) [inline README.md:224]: Copy -> Copy the

File: README.md [hunk 5]
```diff
@@ -207,7 +207,47 @@ owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 0	balance: 0
...
+   `QmRyTc9KbD7ZSkmEf4e7fk6A44RPciW5pM4iyqRGrhbyvj`
+
+3. Execute `tznft mint` command adding IPFS hash as a forth parameter in the token
```
Comment (obstropolos) [inline README.md:227]: Execute -> Execute the 
adding -> adding the

forth -> fourth

File: README.md [hunk 6]
```diff
@@ -207,7 +207,47 @@ owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 0	balance: 0
...
+   description.
+
+```sh
+$ tznft mint bob -t '0, TZT, Tezos Token, QmRyTc9KbD7ZSkmEf4e7fk6A44RPciW5pM4iyqRGrhbyvj'
+
+originating new NFT contract...
+originated NFT collection KT1SgzbcfTtdHRV8qHNG3hd3w1x23oiC31B8
+```
+
+4. Now we can inspect new token metadata and see that IPFS hash (`ipfs_cid`) is
```
Comment (obstropolos) [inline README.md:237]: see that -> see that the

**Review by smondet — APPROVED** [review]

**Pull request merged**